keras/layers/convolutional.py

```diff
@@ -363,26 +363,87 @@ def get_config(self):
                 "size": self.size}
 
 
+class ZeroPadding1D(Layer):
+    """Zero-padding layer for 1D input (e.g. temporal sequence).
+
+    Input shape
+    -----------
+    3D tensor with shape (samples, axis_to_pad, features)
+
+    Output shape
+    ------------
+    3D tensor with shape (samples, padded_axis, features)
+
+    Arguments
+    ---------
+    padding: int
+        How many zeros to add at the beginning and end of
+        the padding dimension (axis 1).
+    """
```

Comment:
I am cool with such a docstring style, it's neat and informative. But then it would need to be matched everywhere else in the repo.